src/QuickActions.ts

```diff
@@ -73,64 +61,95 @@ export class QuickActions {
         return resultsList
     }, this);
 
-    static pushResultUsingPriority(priority: Priority, shortcut: KeyboardShortcut, results: Results) : void {
-        if(priority === Priority.Word){
-            results.word.push(shortcut)
-        }else if(priority === Priority.Tag){
-            results.tag.push(shortcut)
-        }else if(priority === Priority.Start){
-            results.start.push(shortcut)
-        }else if(priority === Priority.TagStart){
-            results.tagStart.push(shortcut)
+    static checkMatch(shortcut: KeyboardShortcut, searchTerm: string): number {
+        const perfectWordMatchPoints = 10000 //search term and shortcut name are a perfect match
```

Comment:
It's just a style I'm used to, but I would rename these constants using ALL_CAPS

And maybe sort them in order of points (descending)?